requirements-cwl.txt

```diff
@@ -6,3 +6,4 @@ ruamel.yaml>=0.15,<=0.19
 ruamel.yaml.clib>=0.2.6
 networkx!=2.8.1,<4
 CacheControl[filecache]
+cwl-utils==0.33
```

Comment:
FYI, this will either need updating with new `cwltool` versions, or relaxing as `cwltool` also depends on `cwl-utils`

https://github.com/common-workflow-language/cwltool/blob/c3bd4ca68c4cb8365528878f21d6827894a33f96/setup.py#L139